### PR TITLE
New Elm Cheat Sheet

### PR DESCRIPTION
**Pull request by pjhampton**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Elm Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Introduces the cheat sheet for the Elm programming language.

**Screenshot 1**
elm1

**Screenshot 2**
elm2

---

Instant Answer Page: https://duck.co/ia/view/elm_cheat_sheet

[Maintainer](https://github.com/pjhampton): @pjhampton

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/elm.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+    "id": "elm_cheat_sheet",
+    "name": "Elm",
+    "description": "An overview of the Elm programming language",
+
+    "metadata": {
+        "sourceName": "Elm Cheat Sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/elm.json]: I think "Github" would be better as source name.

Comment (pjhampton) [inline share/goodie/cheat_sheets/json/elm.json]: :+1: I will update this later

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=elm cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/elm_cheat_sheet_mobile.png)

**Comment by pjhampton** [comment]
@duckduckgo/duckduckhack-contributors Any of you guys have any ideas on improvements or would like to contribute? Would love some feedback from elm users if there is any. 😃

**Comment by GuiltyDolphin** [comment]
foo

**Comment by GuiltyDolphin** [comment]
@pjhampton Thanks! Sorry for the delay :dolphin:

220px-east_coker_elm _2

**Comment by daxtheduck** [comment]
@pjhampton this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by pjhampton** [comment]
Thanks @GuiltyDolphin, and nice tree 😉

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=elm cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/elm_cheat_sheet_mobile.png?CnoDjKB3TL)

**Comment by edgesince84** [comment]
@pjhampton we need to get this added to our Programming Coverage Table once it goes live on duckduckgo.com 💯  cc @tagawa

**Comment by pjhampton** [comment]
Oh, yeah... Thanks for the reminder @edgesince84 😃

**Comment by tagawa** [comment]
Thanks for updating the programming coverage table @pjhampton 

And if there was a contest for best tree photo in a pull request, this would definitely get my vote :-)

**Pull request merged**